src/state_migration/mod.rs

```diff
@@ -109,7 +109,7 @@ where
                 .map(|sr| format!("{}", sr.actors))
                 .unwrap_or_default();
             if new_state != *parent_state {
-                crate::utils::misc::reveal_upgrade_logo(chain_config.network_version(epoch));
+                crate::utils::misc::reveal_upgrade_logo(height.into());
```

Comment:
I'm surprised that `chain_config.network_version(epoch) != height` since `epoch == chain_config.epoch(height)`. I thought those functions would be the inverse of each other.

Reply:
I checked the corresponding logic in lotus and we seem to match each other.  `fn network_version` is widely used in daemon and RPC implementations and I don't think we can easily change it to be the inverse of `chain_config.network_version`

```go
func (sm *StateManager) GetNetworkVersion(ctx context.Context, height abi.ChainEpoch) network.Version {
	// The epochs here are the _last_ epoch for every version, or -1 if the
	// version is disabled.
	for _, spec := range sm.networkVersions {
		if height <= spec.atOrBelow {
			return spec.networkVersion
		}
	}
	return sm.latestVersion
}
```

Reply:
It's even more surprising given that I checked it on my machine (not devnet, though), and it worked as expected. 🤣